looper/looper.py

```diff
@@ -1089,17 +1090,39 @@ def main():
     )
 
     # Initialize project
-    try:
-        p = Project(
-            cfg=args.config_file,
-            amendments=args.amend,
-            divcfg_path=divcfg,
-            runp=args.command == "runp",
-            **{attr: getattr(args, attr) for attr in CLI_PROJ_ATTRS if attr in args},
-        )
-    except yaml.parser.ParserError as e:
-        _LOGGER.error("Project config parse failed -- {}".format(e))
-        sys.exit(1)
+    if is_registry_path(args.config_file):
+        if args.output_dir and (
+            args.pipeline_sample_config or args.pipeline_project_config
+        ):
+            p = Project(
+                amendments=args.amend,
+                divcfg_path=divcfg,
+                runp=args.command == "runp",
+                project_dict=PEPHubClient()._load_raw_pep(
+                    registry_path=args.config_file
+                ),
+                **{
+                    attr: getattr(args, attr) for attr in CLI_PROJ_ATTRS if attr in args
+                },
+            )
+        else:
+            raise MisconfigurationException(
+                f"`pipeline_config` or `output_dir` is missing. Provide it in the parameters."
+            )
+    else:
+        try:
+            p = Project(
+                cfg=args.config_file,
+                amendments=args.amend,
+                divcfg_path=divcfg,
+                runp=args.command == "runp",
+                **{
+                    attr: getattr(args, attr) for attr in CLI_PROJ_ATTRS if attr in args
+                },
```

Comment:
What type of object is `args`? It's a bit odd to have a `getattr` call (`__getattr__`) used in tandem with an `in` check (`__contains__`) since the first is sort of a general object thing and the second is about collection-/container-like things.